demo/vl-typography.html

```diff
@@ -170,6 +169,13 @@ <h6>Heading 6</h6>
                     </tbody>
                 </table>
             </vl-typography>
+        </vl-demo> -->
+
+        <vl-demo data-vl-title="Parameters">
+            <vl-typography data-vl-parameters='{"key1": "tempus", "key2": "ipsum"}'>
+                <p>Lorem <b>${this.key2}</b> dolor sit amet, consectetur adipiscing elit. Duis iaculis molestie feugiat. Lorem <b>${this.key2}</b> eros, consequat et venenatis ac, scelerisque feugiat nunc. Nam molestie tincidunt lectus, nec volutpat ante egestas at. Curabitur quis odio metus. Morbi at purus ac purus convallis <b>${this.key1}</b> at eu est. Nunc id ligula quis justo semper ullamcorper. Donec orci nisi, <b>${this.key1}</b> varius massa ut, vulputate imperdiet nibh. Maecenas <b>${this.key1}</b> lectus quis turpis cursus, ac vehicula ligula fermentum.</p>
```

Comment:
Is 'this' altijd nodig tijdens gebruiken?

Reply:
De oorzaak dat er een `.` gebruikt moet worden is voor CSP. Ik heb `this` vervangen door `parameter` waardoor de eindgebruiker/business minder in contact komt het het technische concept van `this`.

Voorbeeld:
Vroeger: `Dit is een ${this.key1} voorbeeld`
Nu: `Dit is een ${parameter.key1} voorbeeld`